keyboards/keychron/q5/info.json

```diff
@@ -0,0 +1,63 @@
+{
+    "manufacturer": "Keychron",
+    "keyboard_name": "Keychron Q5",
+    "maintainer": "lalalademaxiya1",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "dip_switch": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "rgblight": false
+    },
+    "matrix_pins": {
+        "cols": ["A10", "A9", "A8", "B1", "B0", "A7", "A6", "A5", "A4", "A3", null, null, null, null, null, null, null, null],
+        "custom": true,
+        "custom_lite": true,
+        "rows": ["B5", "B4", "B3", "A15", "A14", "A13"]
```

Comment:
Please leave custom amtrix config as is - col and row pins for custom matrix should not be in info.json as one or both may not actually exist.

Reply:
Seeing how I have this set up for every other board that uses custom matrix (in this PR, the Q6), does that mean having to put them back into `config.h` for custom matrix config?

Reply:
Taken care of in https://github.com/qmk/qmk_firmware/pull/21423/commits/e87a2c37be06c34fbb0ad3045703014537262d00.